fiftyone/server/mutation.py

```diff
@@ -391,6 +408,35 @@ async def set_spaces(
         await dispatch_event(subscription, StateUpdate(state=state))
         return True
 
+    @gql.mutation
+    async def set_color_scheme(
+        self,
+        subscription: str,
+        session: t.Optional[str],
+        dataset: str,
+        stages: BSONArray,
+        color_scheme: ColorSchemeInput,
+        save_to_app: bool,
+        color_scheme_save_format: ColorSchemeSaveFormat,
+    ) -> bool:
+        state = get_state()
+        view = get_view(dataset, stages=stages)
+        state.color_scheme = ColorScheme(
+            color_pool=color_scheme.color_pool,
+            customized_color_settings=color_scheme.customized_color_settings,
+        )
+
+        if save_to_app:
+            view._dataset.app_config.color_scheme = foo.ColorSchemeDocument(
+                color_pool=color_scheme_save_format.color_pool,
+                customized_color_settings=color_scheme_save_format.customized_color_settings,
+            )
+            view._dataset.save()
+            state.view = view
+
+        await dispatch_event(subscription, StateUpdate(state=state))
```

Comment:
you might have to dispatch if save_to_app if you are following save view approach